### PR TITLE
Update raft_configuration.md

### PR DESCRIPTION
**Pull request by sc1000y**
#307 
Signed-off-by: sc1000y <8299176+sc1000y@users.noreply.github.com>

### CODE REVIEW

File: docs/locale/zh_CN/source/raft_configuration.md [hunk 1]
```diff
@@ -75,23 +75,11 @@ Raft 集群要在两个部分进行配置：
...
-   intra-cluster communication.
-   If your cluster is communicating via the same gRPC server that is used
-   to service clients and peers, you need to first reconfigure your orderer
-   by additionally setting `general.cluster.ListenPort`, `general.cluster.ListenAddress`,
-   `ServerCertificate` and `ServerPrivateKey`, and then restarting the orderer
-   in order for the new configuration to take effect.
-
-
+  * `TLSHandshakeTimeShift`:如果ordering节点的TLS证书过期且没有按时更换（参见下面的TLS证书轮换），它们之间就不能建立交流，也不能向ordering服务发送新的交易。
+    为了从这种情况下回复，可以考虑在ordering节点间进行TLS握手并使用`TLSHandshakeTimeShift`来配置一个回溯的时间。
```
Comment (davidkhala) [inline docs/locale/zh_CN/source/raft_configuration.md]: `回复`

File: docs/locale/zh_CN/source/raft_configuration.md [hunk 2]
```diff
@@ -166,21 +154,13 @@ Raft 排序节点支持动态地（意思是，当通道正在使用时）添加
...
-This should be done only under extreme circumstances, where the certificates of the administrators
-have expired, and due to this it is not possible to send configuration updates to replace the administrator
-certificates with renewed ones, because the config transactions signed by the existing administrators
-are now rejected because they have expired.
-After updating the channel it is recommended to change back to the default configuration which enforces
-expiration checks on identities.
+然而，你可以通过打开`General.Authentication.NoExpirationChecks`配置来让orderer忽略过期日期，相关配置在 `orderer.yaml`中。
 
+这仅限于在极端情况下使用，当一个管理员的证书过期，就会导致没有办法发送新的配置文件来更新管理员证书，因为配置文件的签名由当前管理员证书进行，但它因为已经过期而会被拒绝。
+更新通道之后建议将配置文件改回默认配置，即强制标进行识过期检测。
```
Comment (davidkhala) [inline docs/locale/zh_CN/source/raft_configuration.md]: `强制标进行识过期检测`
这是什么？

Comment (sc1000y) [reply docs/locale/zh_CN/source/raft_configuration.md]: After updating the channel it is recommended to change back to the default configuration which enforces
expiration checks on identities. 

强制标识进行过期检测，typo

File: docs/locale/zh_CN/source/raft_configuration.md (hunk 1 above)
Comment (sc1000y) [reply docs/locale/zh_CN/source/raft_configuration.md]: 这几段更改和原文没对应上

Comment (sc1000y) [reply docs/locale/zh_CN/source/raft_configuration.md]: image

Comment (davidkhala) [reply docs/locale/zh_CN/source/raft_configuration.md]: 我的意思是“ 为了从这种情况下回复”应该是`恢复`

Comment (sc1000y) [reply docs/locale/zh_CN/source/raft_configuration.md]: 收到！

**Review by davidkhala — APPROVED** [review]

**Pull request merged**